DEVELOPER.md

```diff
@@ -190,106 +182,48 @@ To inform the application what kind of command your autotest firmware
 can handle, you must write the name of the test in the list
 `AUTOTEST_AVAILABLE` as shown in the template file.
 
+You can find the code (based on RIOT operating system) for the autotest and idle
+firmware for some boards supported on IoT-LAB
+on [ci-firmwares](https://github.com/iot-lab/ci-firmwares)
+
 Testing your implementation
 ---------------------------
 
 ### Unit tests
 
-When developing your implementation, you should verify that all unit tests pass on your computer by running `tox`.
-
-> You need an up to date `tox` version. I recommend installing it with pip
->
->     sudo -H pip install --upgrade tox
-
-### Integration tests
+To run these tests, you will need the liboml dependency, see [INSTALL.md](INSTALL.md)
 
-To run the integration/system tests directly on the board, including the autotest use the following command:
+When developing your implementation, you should verify that all unit
+tests pass on your computer by running `tox`.
 
-    fab -f tests_utils/integration_fabfile.py python_test -H adress-of-your-node
-
-> It requires also quite up to date fabric version, I'm running `fabric 1.10.2` with `paramiko 1.15.2`
+> If you don't have tox, install it with
 >
->     sudo -H pip install --upgrade fabric
-
-
-Annexe
-======
-
-template file
--------------
+>     pip install tox
 
-Bellow you can find a template file which interface with the gateway
-application with proper comment, free to you to use it and modify it:
+### Integration tests (locally via Docker)
 
-    # -*- coding:utf-8 -*-
-    """ Blank file for the implemention of an open-node called Nodename """
+The purpose of integration tests is verifying that your firmwares
+ behave as expected when called by the gateway code.
 
-    from gateway_code.config import static_path
-    from gateway_code import common
-    from gateway_code.common import logger_call
+We've provided Dockerfiles for the dependencies and a Makefile
+for you to build and use them. (if using MacOS, see Annex below)
 
-    import logging
-    LOGGER = logging.getLogger('gateway_code')
+Build the Docker images (`Dockerfile` and `tests\Dockerfile`) through
 
+   make build-docker-image build-docker-image-test
```

Comment:
`make build-docker-image-test` is enough (there's a dependency on `build-docker-image`)